src/wordpiece_tokenizer.cpp

```diff
@@ -94,19 +101,40 @@ bool WordpieceTokenizer::evaluate(ov::TensorVector& outputs, const ov::TensorVec
         new_begins[seq] = ragged_offset;
 
         for(size_t ragged_col = ragged_begins[seq]; ragged_col < ragged_ends[seq]; ++ragged_col) {
-
-            auto str = std::string(chars + begins[ragged_col], chars + ends[ragged_col]);
-            std::vector<core::Token> results = m_tokenizer->Tokenize(str);
-
-            for (const core::Token& token : results) {
+            
```

Comment:
We can check if the input size is bigger than `m_max_bytes_per_word` here and return `[UNK]` token if it is.

Reply:
done